BaseStation/MainWindow/MainWindow.xaml.cs

```diff
@@ -46,7 +49,10 @@ public MainWindow()
                 }
             }
             Map.DisplayMap(Settings.CurrentMapFile);
-		}
+            Map.Waypoints = Waypoints;
+            ArmSideViewer.ViewLabel.Content = "Arm Side";
```

Comment:
Can this be moved into the `ArmSideView` and `ArmTopView` classes? Maybe through an abstract property (if that's possible) or an abstract method.

Reply:
not really sure how